src/components/ContentSection.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+import EventCard from './EventCard'
+import { EventsListType } from '~/lib/sanity.interfaces'
+
+export default function ContentSection({ events }: EventsListType) {
+  console.log('events n component', events, typeof events)
+  //sort - > cards by date;
+  return (
+    <section className="py-16 ">
+      <h2 className="font-bold text-center text-[42px]  text-[#21272A] font-roboto ">
+        Event Title
+      </h2>
+      <p className="text-center font-roboto color-[#21272A] mt-4 font-normal text-lg  mb-[3.75rem]">
+        Paragraph. Lorem ipsum dolor sit amet consectetur adipisicing elit.
```

Comment:
This text as well should be dynamic and come from schema as for example `intro` or `description`